internal/metrics/Metrics.go

```diff
@@ -18,6 +22,26 @@ func (g Gauge) GetRaw() float64 {
 	return float64(g)
 }
 
+func (g Gauge) Value() (driver.Value, error) {
+	return g.GetRaw(), nil
+}
+
+func (g *Gauge) Scan(value interface{}) error {
```

Comment:
Сейчас есть алиас `any` для `interface{}`, мне кажется он предпочтительней.
```suggestion
func (g *Gauge) Scan(value any) error {
```

Reply:
Поправил